common/src/main/java/com/wynntils/gui/screens/settings/WynntilsBookSettingsScreen.java

```diff
@@ -261,8 +266,11 @@ private void scrollFeatureList(double delta) {
     }
 
     private void scrollConfigList(double delta) {
-        configScrollOffset =
-                MathUtils.clamp((int) (configScrollOffset - delta), 0, Math.max(0, configs.size() - CONFIGS_PER_PAGE));
+        int roundedUpPageNeed = configs.size() / CONFIGS_PER_PAGE + (configs.size() % CONFIGS_PER_PAGE == 0 ? 0 : 1);
```

Comment:
This is static, cache it

Reply:
Same, I think it might create mess if cached.